rockcraft.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+package-repositories:
+  - components: [main]
+    key-id: 0DCFCA5547B19D2A6099506096B3EE5F29111145
+    key-server: https://packages.wazuh.com/key/GPG-KEY-WAZUH
+    suites: [stable]
+    type: apt
+    url: https://packages.wazuh.com/4.x/apt/
```

Comment:
Is this pulling in build dependencies? I think we want to avoid using any external binaries (but this can happen in a later stage, if we leave this, we should add a note in a comment about it).

Reply:
It's a bit tricky, so I'm planning that for a follow up PR